geomstats/geometry/base.py

```diff
@@ -320,32 +320,33 @@ class LevelSet(Manifold, abc.ABC):
     ----------
     dim : int
         Dimension of the embedded manifold.
-    embedding_space : VectorSpace
-        Embedding space.
     default_coords_type : str, {'intrinsic', 'extrinsic', etc}
         Coordinate type.
         Optional, default: 'intrinsic'.
     """
 
-    def __init__(
-        self,
-        dim,
-        embedding_space,
-        submersion,
-        value,
-        tangent_submersion,
-        default_coords_type="intrinsic",
-        **kwargs
-    ):
-        kwargs.setdefault("shape", embedding_space.shape)
+    def __init__(self, dim, default_coords_type="intrinsic", **kwargs):
```

Comment:
I think the default should be extrinsic

Reply:
Completely agree! (I think we have inconsistencies in some parts of the library due to this) Later I guess we are changing this to `intrisinc = False` (see #1725).